config/routes.rb

```diff
@@ -395,15 +394,24 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
-      get '/by_mail' => 'gpo_verify#index', as: :gpo_verify
-      post '/by_mail' => 'gpo_verify#create'
+      get '/by_mail/enter_code' => 'by_mail/enter_code#index', as: :gpo_verify
+      post '/by_mail/enter_code' => 'by_mail/enter_code#create'
       get '/by_mail/confirm_start_over' => 'confirm_start_over#index',
           as: :confirm_start_over
 
       if FeatureManagement.gpo_verification_enabled?
-        get '/usps' => 'gpo#index', as: :gpo
-        put '/usps' => 'gpo#create'
+        get '/by_mail/request_letter' => 'by_mail/request_letter#index', as: :gpo
+        put '/by_mail/request_letter' => 'by_mail/request_letter#create'
       end
+
+      get '/by_mail/letter_enqueued' => 'by_mail/letter_enqueued#show', as: :gpo_letter_enqueued
+
+      # Temporary redirects etc. to support GPO route renaming in the 50/50 state
+      get '/by_mail' => redirect('/verify/by_mail/enter_code')
+      post '/by_mail' => 'by_mail/enter_code#create'
+      get '/usps' => redirect('/verify/by_mail/request_letter')
+      put '/usps' => 'by_mail/request_letter#create' if FeatureManagement.gpo_verification_enabled?
```

Comment:
Why add the feature flag check just on put? I would think you would want it on get if anything.